mathesar/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+# We need to do this to register the model correctly in Django settings
+from .users import User  # noqa
```

Comment:
Question: Does this indicate it would be better to move models back into the default `models.py` file?

Reply:
I don't think so – this is specific to the `User` model and Django's "swappable user model" architecture. None of the other models need to be imported here.